src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,35 @@
-export const PostInfo = () => <>Put the post here</>;
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+
+import usersFromServer from '../../api/users.json';
```

Comment:
Ensure that the `users.json` file exists in the `api` directory and contains the necessary user data for the `UserInfo` component.